src/qat/purr/integrations/qasm.py

```diff
@@ -1995,12 +2001,36 @@ def _capture_iq_value(
         # these post processing operations are removed for executions on live hardware.
         #
         # The returned value for each shot after postprocessing is a complex iq value.
+
+        # Determine the delay for the channel
+        delay = 0.0
+        if pulse_channel.channel_type == ChannelType.acquire:
+            # TODO: remove full id when previous PR is merged
```

Comment:
Is this TODO something you can action now?

Reply:
Will action in #202 :smile: